homeassistant/components/proxmoxve/__init__.py

```diff
@@ -11,41 +11,44 @@
 from requests.exceptions import ConnectTimeout, SSLError
 import voluptuous as vol
 
+from homeassistant.config_entries import SOURCE_IMPORT, ConfigEntry
 from homeassistant.const import (
     CONF_HOST,
     CONF_PASSWORD,
     CONF_PORT,
     CONF_USERNAME,
     CONF_VERIFY_SSL,
-    Platform,
 )
 from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryAuthFailed
+from homeassistant.helpers import device_registry
 import homeassistant.helpers.config_validation as cv
-from homeassistant.helpers.discovery import async_load_platform
+from homeassistant.helpers.entity import DeviceInfo
 from homeassistant.helpers.typing import ConfigType
 from homeassistant.helpers.update_coordinator import (
     CoordinatorEntity,
     DataUpdateCoordinator,
 )
 
-PLATFORMS = [Platform.BINARY_SENSOR]
-DOMAIN = "proxmoxve"
-PROXMOX_CLIENTS = "proxmox_clients"
-CONF_REALM = "realm"
-CONF_NODE = "node"
-CONF_NODES = "nodes"
-CONF_VMS = "vms"
-CONF_CONTAINERS = "containers"
-
-COORDINATORS = "coordinators"
-API_DATA = "api_data"
-
-DEFAULT_PORT = 8006
-DEFAULT_REALM = "pam"
-DEFAULT_VERIFY_SSL = True
-TYPE_VM = 0
-TYPE_CONTAINER = 1
-UPDATE_INTERVAL = 60
+from .const import (
+    CONF_CONTAINERS,
+    CONF_LXC,
+    CONF_NODE,
+    CONF_NODES,
+    CONF_QEMU,
+    CONF_REALM,
+    CONF_VMS,
+    COORDINATORS,
+    DEFAULT_PORT,
+    DEFAULT_REALM,
+    DEFAULT_VERIFY_SSL,
+    DOMAIN,
+    INTEGRATION_NAME,
+    PLATFORMS,
+    PROXMOX_CLIENT,
+    UPDATE_INTERVAL,
+    ProxmoxType,
+)
 
 _LOGGER = logging.getLogger(__name__)
```

Comment:
Another standard: if you log throughout this integration, you could define it once (in `const.py`):

```suggestion
LOGGER = logging.getLogger(__package__)
```